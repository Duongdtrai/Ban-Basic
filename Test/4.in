34
92
93 79 92 42 26 87 71 73 30 66 56 76 96 97 94 86 46 67 16 85 44 52 80 9 34 38 94 89 69 21 31 52 76 98 18 43 39 95 43 14 6 89 32 69 18 38 24 34 61 30 54 69 100 33 77 36 77 15 56 21 11 90 1 42 62 100 94 47 31 6 24 10 8 12 89 37 88 69 15 59 63 30 62 34 75 40 100 5 16 8 42 92 
31
98 19 16 97 29 6 68 22 74 23 19 6 97 12 87 63 76 72 88 39 25 8 31 79 64 100 15 71 76 46 45 
4
79 91 27 62 
85
9 28 94 48 1 83 1 33 72 30 74 76 8 9 93 49 62 75 85 69 55 47 47 1 24 74 82 25 39 68 67 43 85 35 42 91 84 70 73 100 26 69 11 52 65 42 96 78 36 3 61 87 55 92 97 22 16 76 56 25 30 73 26 80 81 28 55 97 91 45 52 38 97 12 34 91 65 55 18 57 1 94 1 57 8 
99
97 45 74 43 58 16 23 1 8 8 34 26 21 51 23 15 60 17 36 5 97 91 79 64 84 18 23 91 32 98 72 70 75 23 1 71 93 8 35 54 63 16 32 96 84 87 48 15 76 56 87 39 29 71 76 89 12 44 93 84 100 83 98 56 50 27 18 45 65 90 56 58 92 59 4 3 14 17 13 64 24 86 67 90 96 94 34 52 81 6 52 17 100 96 82 77 31 53 16 
17
29 20 8 7 91 4 88 69 6 85 44 11 17 87 67 82 14 
98
87 78 40 8 12 47 90 82 49 52 66 53 47 64 100 37 3 98 45 77 36 50 72 55 11 64 85 45 45 77 40 3 10 58 32 17 40 40 63 60 74 58 36 79 94 93 50 11 98 56 57 1 15 65 64 53 8 27 85 34 60 2 29 46 16 24 14 48 80 46 54 19 59 50 81 31 67 44 93 47 67 43 13 2 76 56 31 14 24 71 6 48 11 48 57 91 51 94 
95
20 33 19 2 97 55 70 53 83 83 56 76 4 79 63 27 81 59 65 16 18 76 38 43 78 95 83 41 62 10 3 72 70 28 32 83 17 44 69 44 92 12 88 100 71 32 4 72 5 64 24 5 20 8 50 93 48 3 59 74 5 16 41 56 24 47 5 77 62 47 63 73 91 50 69 87 97 38 81 56 16 43 59 72 55 57 96 30 32 55 44 3 78 8 50 
13
7 19 25 69 7 45 75 94 21 79 60 13 66 
31
30 99 65 57 63 8 86 70 42 7 39 23 53 39 83 98 70 98 37 12 15 87 20 89 53 49 84 64 29 75 16 
41
16 23 59 61 96 31 62 90 84 8 73 14 19 28 76 49 22 54 95 21 33 98 57 28 66 48 58 64 80 54 57 60 31 37 65 80 28 17 21 84 72 
87
98 62 59 70 88 82 32 20 64 82 56 39 44 13 86 72 93 73 85 79 4 72 3 5 90 100 70 67 79 8 4 58 36 82 23 53 93 71 97 35 35 47 7 75 59 90 88 79 82 95 94 46 91 64 24 22 55 35 65 22 43 91 84 23 34 41 77 55 77 66 12 5 11 75 100 1 54 28 57 93 77 99 72 31 16 86 18 
21
82 55 26 28 76 48 87 100 57 24 50 4 9 15 4 65 92 86 4 42 100 
34
97 68 23 98 20 27 88 53 14 6 46 61 21 48 1 67 31 69 72 31 21 48 16 98 10 43 61 12 52 58 73 3 39 25 
56
72 36 37 48 25 63 17 93 63 16 14 46 87 30 9 53 61 31 65 44 22 58 34 76 15 99 34 79 40 86 98 92 73 68 88 19 23 10 85 17 19 1 89 99 97 47 29 66 59 41 46 46 18 76 8 9 
32
66 85 45 94 59 24 77 69 48 82 75 5 17 26 24 14 10 23 83 64 68 3 100 7 27 19 42 46 58 1 28 70 
57
35 89 91 75 42 96 36 91 64 59 57 56 23 86 12 8 62 71 85 13 21 15 8 71 66 11 50 4 65 32 24 7 2 8 73 96 45 31 28 73 80 87 40 56 65 55 98 80 97 7 76 54 98 87 64 65 29 
8
4 1 35 49 1 22 28 86 
96
45 52 97 15 44 66 17 53 94 3 53 8 9 28 92 43 1 58 51 90 3 4 72 93 31 97 26 8 12 68 95 2 96 59 28 24 49 4 6 20 69 30 50 66 39 42 26 64 78 42 8 56 98 58 96 68 84 39 86 39 52 8 84 86 31 82 70 76 91 8 2 31 11 1 15 21 99 53 90 66 49 22 38 63 65 19 36 84 93 42 52 15 91 69 11 54 
97
21 40 66 70 81 81 48 67 55 94 74 4 31 72 92 83 27 81 23 80 78 65 16 97 99 62 25 57 29 61 50 22 53 6 92 90 49 86 65 63 7 57 14 38 52 23 4 97 68 12 51 41 5 62 60 45 95 45 62 66 78 82 58 27 90 97 91 72 33 83 25 44 27 62 8 41 67 46 28 74 98 64 9 54 18 82 70 37 52 95 91 67 38 31 9 85 79 
18
1 60 18 70 41 42 78 16 90 69 18 40 97 7 14 13 1 2 
54
8 25 23 39 82 39 56 69 34 80 3 52 48 13 51 16 43 49 98 97 23 69 72 74 55 37 29 41 63 72 55 84 100 18 30 82 44 29 97 16 35 62 10 93 85 68 49 34 28 36 54 21 10 9 
70
95 84 55 9 93 39 38 41 50 97 59 94 97 93 51 18 2 38 67 59 75 56 86 24 7 51 50 94 95 22 90 99 88 4 27 76 47 56 84 54 99 64 63 27 64 19 31 88 58 31 15 21 48 52 27 85 46 94 55 70 54 65 62 14 87 2 56 73 82 13 
19
11 82 62 12 60 67 73 66 93 39 95 53 79 32 31 39 97 52 34 
98
96 36 68 16 10 38 32 17 32 72 99 77 8 84 8 19 9 39 21 27 79 6 72 66 86 42 81 32 45 66 66 27 21 53 3 14 17 96 8 75 92 44 16 74 72 4 56 29 78 93 58 60 85 80 57 25 41 8 69 84 25 59 15 54 55 37 99 24 88 29 75 79 77 86 24 46 56 2 23 51 37 26 11 60 65 37 16 64 34 98 76 29 59 89 18 66 67 1 
32
87 96 43 79 71 72 1 39 1 31 28 6 11 98 77 100 97 55 44 64 83 41 2 44 21 60 26 18 84 40 89 82 
33
78 22 63 33 57 99 4 11 17 7 41 46 78 55 27 21 34 1 100 14 2 2 88 5 65 12 8 87 100 77 58 76 30 
84
8 19 17 53 18 7 67 93 5 95 30 72 34 63 41 81 62 27 42 93 62 32 24 59 13 42 83 76 93 77 18 15 22 83 79 44 35 96 53 89 72 64 52 12 67 55 18 37 59 56 58 90 91 61 98 4 78 26 81 4 78 5 2 90 6 99 94 58 67 35 5 93 67 31 63 35 66 72 41 80 23 29 57 97 
93
6 5 1 63 41 83 92 80 5 31 60 11 32 20 7 77 44 49 37 71 4 61 31 14 56 24 18 19 36 21 88 64 77 72 59 41 19 71 12 16 87 78 47 84 3 53 27 24 63 42 27 37 79 100 65 98 56 37 61 8 18 62 18 84 32 90 1 61 21 1 49 72 35 74 37 25 6 18 14 33 21 26 40 74 83 23 9 5 66 30 28 37 42 
9
93 80 71 94 26 84 88 1 46 
92
85 28 54 77 33 15 1 33 88 75 54 71 61 43 32 60 21 100 43 99 47 68 36 56 11 50 3 62 74 63 8 37 40 6 22 46 87 68 27 60 56 79 11 52 58 35 66 84 48 36 69 10 84 38 91 67 25 27 11 19 38 47 67 36 17 62 92 97 71 68 41 37 64 87 26 9 96 81 3 44 86 41 49 67 49 63 49 73 98 59 89 31 
89
80 94 85 8 68 71 82 38 40 61 77 50 65 28 59 38 93 86 70 23 93 31 36 12 62 64 80 61 29 5 50 95 43 17 33 62 97 7 89 48 40 85 66 55 78 70 26 99 97 90 79 32 54 88 86 70 15 89 12 5 25 26 24 96 17 58 54 32 46 9 56 67 55 96 54 79 16 87 21 86 87 57 84 92 87 55 42 11 37 
61
75 97 80 18 70 60 83 39 16 85 37 77 75 33 57 19 78 34 17 24 86 48 54 77 89 61 100 82 45 16 2 17 23 79 17 83 59 10 91 39 13 14 69 18 85 77 97 84 49 93 41 8 40 97 63 88 15 57 7 28 84 
2
37 5 
