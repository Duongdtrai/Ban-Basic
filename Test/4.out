1005005000
151623314
14958
9007582766
18032560728
10470742
6328718448
12365841554
4098744
149626344
463301510
5308791484
26429798
131952120
1453837686
15954840
1863075608
104060
12759074942
8111972964
5808512
593323288
2125727760
11280414
6919403546
23896404
68922610
9017042146
7913915904
901038
3115858016
11860761178
1150541076
0
